docs/getting_started/dashboard/dashboard_block_reference.md

```diff
@@ -111,11 +111,19 @@ On the **Design** tab, in the **View** field, select the layout to be used to pr
 
 ## Recent activity block [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]
 
-Displays a list of recent activity of all or selected users.
-It also includes a link to view all activities available in Admin tab.
+Displays recent activity of all or selected users, and for selected object classes. Lists the following details:
+
+- action time
+- user reference (avatar, first and last name) with a link to the user profile (if available)
+- activity type with context
```

Comment:
I prefer having only the settings as a list, it help to skim the reference to have this visual pattern.